src/diffusers/configuration_utils.py

```diff
@@ -358,8 +359,21 @@ def load_config(
                 "`self.config_name` is not defined. Note that one should not load a config from "
                 "`ConfigMixin`. Please make sure to define `config_name` in a class inheriting from `ConfigMixin`"
             )
-
-        if os.path.isfile(pretrained_model_name_or_path):
+        # Custom path for now
+        if dduf_entries:
+            if subfolder is not None:
+                raise ValueError("DDUF file only allow for 1 level of directory. Please check the DDUF structure")
```

Comment:
Can we please supplement this with an example in the comments?

Reply:
You mean add an example of path that doesn't work ?

Reply:
Exactly.